customcontext/context.cpp

```diff
@@ -616,9 +572,13 @@ QQuickTextureFactory *Context::createTextureFactory(const QImage &image)
 
 
 
-void CONTEXT_CLASS::renderNextFrame(QSGRenderer *renderer, GLuint fbo)
+#if QT_VERSION >= 0x050800
+void RenderContext::renderNextFrame(QSGRenderer *renderer, uint fbo)
```

Comment:
This looks interesting too. So old version was with latter parameter QLuint, which is still happening below, but the header has also QLuint for >5.14 and uint version which didn't exist before. Or should it just be QLuint for everything?

Reply:
Looking at git version and history, no. GLuint was before 5.8 and from there on it's uint. So this is ok here, but the header as it wrong way. Also as 5.14 didn't have any changes there, could perhaps have it's own #if handling there.